packages/extension/src/ui/index.tsx

```diff
@@ -33,8 +33,8 @@ const root = createRoot(container)
 
 root.render(
   <StrictMode>
-    <BrowserRouter>
+    <HashRouter>
```

Comment:
> Routing in browser has a bug that on refresh the app lose the route

Could you please give me a bit more info about the issue? Do you mean refreshing the extension wallet?